### PR TITLE
Updated README, and added a Getting started guide

### PR DESCRIPTION
**Pull request by breakreturn**
Signed-off-by: Joakim Gross <joakim.gross@pelagicore.com>

### CODE REVIEW

File: getting-started.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+Getting started
+===============
+This section describes how to get started using a PELUX basline build.
```
Comment (GordanM) [inline getting-started.md]: Nitpick: ... using a PELUX **baseline** build.

File: getting-started.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+The general procedure involves writing the built image to e.g. a USB-stick
+or SD-card, and booting up a target using that. The following sections are divided
```
Comment (GordanM) [inline getting-started.md]: Nitpick: no comma before and in this case.

File: getting-started.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+* In that top level directory there is a directory named `build`
+* The `pelux-intel.xml` manifest was used
+* The `core-image-pelux` image was built
+
+Writing the image
+-----------------
+Poky provides a conveniece script for writing images. Assuming using an unmounted
+USB-stick or SD-card with an ext3 partition, do:
+
+```$HOME/pelux-baseline/sources/poky/scripts/contrib/mkefidisk.sh <host-device> $HOME/pelux-baseline/build/tmp/deploy/images/intel-corei7-64/core-image-pelux-intel-corei7-64.hddimg <target-device>```
```
Comment (DunderRoffe) [inline getting-started.md]: I would prefer keeping the "```" on separate rows.

File: getting-started.md (hunk 1 above)
Comment (thenor) [inline getting-started.md]: Typo: "basline" -> "baseline"

File: getting-started.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+Getting started
+===============
+This section describes how to get started using a PELUX basline build.
+
+The general procedure involves writing the built image to e.g. a USB-stick
+or SD-card, and booting up a target using that. The following sections are divided
+in the steps involved, and have sub-sections where there is a difference
+between the hardware targets.
+
+These instructions assumes there is an image built according to the
```
Comment (GordanM) [inline getting-started.md]: These instructions **assume** ...

File: getting-started.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+[README.md](README.md), with the following additional assumptions:
+
+* The top level of the build is located in `$HOME/pelux-baseline`
+* In that top level directory there is a directory named `build`
+* The `pelux-intel.xml` manifest was used
+* The `core-image-pelux` image was built
+
+Writing the image
+-----------------
+Poky provides a conveniece script for writing images. Assuming using an unmounted
```
Comment (thenor) [inline getting-started.md]: Typo: "conveniece" -> "convenience"

Comment (thenor) [inline getting-started.md]: I would also prefer a slight rephrasing of the last sentence. E.g. "If using a USB-stick or SD-card with an ext3 partition, do".

File: getting-started.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+```$HOME/pelux-baseline/sources/poky/scripts/contrib/mkefidisk.sh <host-device> $HOME/pelux-baseline/build/tmp/deploy/images/intel-corei7-64/core-image-pelux-intel-corei7-64.hddimg <target-device>```
+
+Where `<host-device>` is the device on the host, e.g. `/dev/sdc`, and `<target-device>` is how the target hardware will see the media, e.g. `/dev/sda`.
+
+Booting on target hardware
+--------------------------
+
+### NUC
+Enter the USB-stick and boot the NUC. No other actions should be needed.
```
Comment (thenor) [inline getting-started.md]: "Plug in" instead of "Enter" perhaps.

File: getting-started.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+* The top level of the build is located in `$HOME/pelux-baseline`
+* In that top level directory there is a directory named `build`
+* The `pelux-intel.xml` manifest was used
+* The `core-image-pelux` image was built
+
+Writing the image
+-----------------
+Poky provides a conveniece script for writing images. Assuming using an unmounted
+USB-stick or SD-card with an ext3 partition, do:
```
Comment (GordanM) [inline getting-started.md]: Is the ext3 partition mandatory in this step? Isn't the whole device repartitioned and overwritten?

Comment (DunderRoffe) [inline getting-started.md]: I think @GordanM is right. The content of the USB-stick prior to copying the image should not matter.

Comment (breakreturn) [inline getting-started.md]: Yeah the script seems to repartition.

**Pull request merged**